### PR TITLE
SPOI-11438: Adding operator documentation for drools

### PR DESCRIPTION
**Pull request by DT-Priyanka**

### CODE REVIEW

File: docs/operators/droolsRuleExecutor.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+Drools Rules Execution Operator
+=====================
+
+### Introduction
+This operator processes incoming tuples against set of rules to identify meaningful events which can be acted upon quickly. Drools rules execution operator uses apache licensed [drools](https://www.drools.org/) librabry to do rules processing.
+
+
+### Why is it needed ?
```
Comment (jaysingbahirat) [inline docs/operators/droolsRuleExecutor.md]: Why it is needed?

File: docs/operators/droolsRuleExecutor.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+Drools Rules Execution Operator
+=====================
+
+### Introduction
+This operator processes incoming tuples against set of rules to identify meaningful events which can be acted upon quickly. Drools rules execution operator uses apache licensed [drools](https://www.drools.org/) librabry to do rules processing.
```
Comment (jaysingbahirat) [inline docs/operators/droolsRuleExecutor.md]: Replace 'librabry' with 'library'

File: docs/operators/droolsRuleExecutor.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+
+### Why is it needed ?
+Many business domains need to process data to identiy meaningful events or patterns to take certian actions. Those businesses will need this operator.
```
Comment (jaysingbahirat) [inline docs/operators/droolsRuleExecutor.md]: Replace 'identiy' with 'identify' and 'certian' with 'certain'

File: docs/operators/droolsRuleExecutor.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+#### Ports
+factsInput  &lt;Object&gt;: Receives tuples from upstream operators.
+
+factsOutput &lt;Object&gt;: Emits processed facts on factsOutput port.
+
+ruleCountOutput &lt;Map<String, MutableInt>&gt;: Map of rules and number of times each rule was fired in a window.
+
+firedRuleAndTransactionOutput &lt;KeyValPair<String, Set<Object>>&gt;: Map of rules and list of tuples wich triggered the rule in window.
+
```
Comment (jaysingbahirat) [inline docs/operators/droolsRuleExecutor.md]: Replace 'wich' with 'which'

File: docs/operators/droolsRuleExecutor.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+<col width="25%" />
+<col width="75%" />
+<tbody>
+<tr class="odd">
+<td align="left"><p>Parameter</p></td>
+<td align="left"><p>Description</p></td>
+</tr>
+<tr class="even">
+<td align="left"><p>rulesDir</p></td>
+<td align="left"><p>HDFS path of rule files/direcotory containing files. Supported format are drl and decision tables (xls). <br/>If rulesDir is not configured default packaged rules are loaded by operator.</p></td>
```
Comment (jaysingbahirat) [inline docs/operators/droolsRuleExecutor.md]: replace 'direcotory' with 'directory'

**Comment by lizygeogy** [comment]
@DT-Priyanka 
I guess this one is redundant for now. Please verify and reply.

**Comment by lizygeogy** [comment]
verified with @DT-Priyanka 
Closing this PR

**Pull request closed without merging**